### DISCUSSION TITLE
Calling the Places API legacy instead of Places API (NEW)

### DISCUSSION THREAD

**Discussion by imanezouhour** [visgl/react-google-maps#508] [Q&A] [answered]
I 'm using useMapLibrary('places') to use the Places API, so I enabled  Places API (NEW) on my google cloud console account, and I was getting an error saying DENIED_REQUEST, I enabled Places API (LEGACY) on my google cloud console and the request to the Places API is working, but I noticed that the call is sent to the Places API (Legacy).

I need some clarifications on this, How to force the use for the Places API (NEW) ?
Thank you.

**Comment by usefulthink** [comment, marked as answer]
Have a look here: https://github.com/visgl/react-google-maps/issues/465#issuecomment-2272854359

Essentially: The two will likely coexist for a bit, if you want to use the new API backend, you also need to use the new API classes (`google.maps.places.Place`, `google.maps.places.AutocompleteSuggestion` etc.). There's no compatibility of the older API classes with the new backend service.
